src/subscription_manager/utils.py

```diff
@@ -608,17 +608,26 @@ def get_process_names():
             match = re.search(proc_name_expr, lines)
             if match:
                 proc_name = match.groupdict().get("proc_name")
+                try:
+                    proc_id = int(subdir)
+                except ValueError:
+                    proc_id = None
```

Comment:
this check is not needed: since this is already inside `if re.match("[0-9]+", subdir):`, then it will be always an integer value